webapp/src/components/singlestudy/NavHeader.tsx

```diff
@@ -183,6 +202,22 @@ function NavHeader(props: PropTypes) {
     navigate("/studies");
   };
 
+  const copyId = (): void => {
+    if (study) {
+      try {
+        navigator.clipboard.writeText(study.id);
```

Comment:
You need to add async/await because `navigator.clipboard.writeText()` returns a Promise.